src/LanguageMapper.mjs

```diff
@@ -39,7 +39,11 @@ export const getLanguageByNynorskName = name => mappedValue(
 )
 
 export const getLanguageBySamiName = name => mappedValue(
-  ALL_LANGUAGES.find(language => isSameNaturalLanguageString(language.sme.toLowerCase(), convertString(name)))
+  ALL_LANGUAGES.find(language => isSameNaturalLanguageString(language.sme ? language.sme.toLowerCase() : '', convertString(name)))
+)
+
+export const getLanguageByIso6395Code = code => mappedValue(
```

Comment:
tolker dette som at tanken er at språkfeltet kan veksle mellom ISO kode `6393` og `6395` som verdi nå fremover?

Reply:
ja

Reply:
Vet ikke hvor mye dere benytter disse utils klassene på frontend, men i backend har vi en utils metode som heterGetLanguageByPotentialIsoCode. Da får du ut et språk uavhengig om det er 1,2, 3 eller 5 varianten av iso-koden.
Kan implementere den her også hvis du ønsker.

Reply:
I frontend brukes den egentlig bare for å populere språk-dropdown: https://github.com/BIBSYSDEV/NVA-Frontend/blob/bb986928fa62151b0fd9e649ae19bbca2371d78e/src/utils/registration-helpers.ts#L733

Tenkte først det ville bli ulike IDer her, og at vi måtte sjekke for likhet mot begge ISO-kodene, men etter å ha sett litt rundt ser det ut som vi bare bruker `uri`, så da vil jeg tro det ikke trengs noen endringer likevel